GUI/QtQuick/ArmoryServersModel.cpp

```diff
@@ -70,11 +72,9 @@ void ArmoryServersModel::setData(int curIdx, int connIdx
 
 void ArmoryServersModel::add(const ArmoryServer& srv)
 {
-   QMetaObject::invokeMethod(this, [this, srv] {
-      beginInsertRows(QModelIndex{}, rowCount(), rowCount());
-      data_.push_back(srv);
-      endInsertRows();
-      });
+   beginInsertRows(QModelIndex{}, rowCount(), rowCount());
+   data_.push_back(srv);
+   endInsertRows();
```

Comment:
What's the intention here? I'm not sure that I've seen the `invokeMethod` wrapping instead in QtQuickAdapter.